modules/calendar.xml

```diff
@@ -183,6 +188,12 @@
     <onParseData><![CDATA[
 // item - element to be parsed
 // properties - object containing the values to be evaluated/used
+
+// If item comes as empty from data, skip parsing
+if (item === null) {
```

Comment:
The flow diagram doesn't have on it a case where onParseData can return null - is this a mistake or is the flow diagram incorrect?

Reply:
Should those values be filtered in `onDataLoad` instead @dasgarner ? 🤔

Reply:
It makes logical sense to me that if you want to remove data you do it in onDataLoad, and we make it so that onParseData must return an item (and if it doesn't we fail safe and use the original)?

Reply:
Yeah, I think that makes total sense. I already moved that filtering logic to `onDataLoad` and it's working :)